api/api.go

```diff
@@ -1012,6 +1028,20 @@ func (serv *MetadataServer) CreateSourceVariant(ctx context.Context, sourceReque
 	return serv.meta.CreateSourceVariant(ctx, sourceRequest)
 }
 
+func extractSourcesFromSQLTransformation(query string) []*pb.NameVariant {
```

Comment:
this is a good candidate for parametrized unit tests if they don't already exist.